### ISSUE TITLE
branch: 1.1-phetio

### ISSUE THREAD

**Issue by pixelzoom** [phetsims/function-builder#140]
In https://github.com/phetsims/function-builder/issues/139, I was attempting to publish a new version of Function Builder.  The published version is 1.0.x, so I was expecting to be publishing 1.1.x. But when I went to publish the dev version of Function Builder, it was 1.2.x. Investigation revealed a 1.1-phetio branch and many 1.1 versions published to bayes.     That resulted in a lot of confusion and this significant detour.

It looks like @samreid, @zepumph, and @jonathanolson were involved in this. As the responsible dev for this repo,  please help me understand (or remember) what is going on here. 

There is no 1.1 branch, but there is a 1.1-phetio branch. I'm not sure who created it.

function-builder has no PhET-iO instrumentation, and brand=phet is not enabled in package.json.

Here are the 1.1 versions on bayes, annotated with who published them (accordiong to git history):

```
% ls -ltd 1.1.*
drwxrwsr-x+ 2 tomcat     phet 118 Jun  5  2017 1.1.0-dev.4/ => ??
drwxrwsr-x+ 2 tomcat     phet 102 May 17  2017 1.1.0-dev.3/ => ??
drwxrwxr-x+ 5 tomcat     phet 133 Nov 12  2016 1.1.2-phetiophetio/ => @samreid 
drwxrwxr-x+ 5 tomcat     phet 134 Oct  5  2016 1.1.1-phetio/ => @zepumph 
drwxrwxr-x+ 6 tomcat     phet 151 Sep 30  2016 1.1.0-phetio/ => @zepumph 
drwxrwsr-x+ 5 tomcat     phet 207 Sep 28  2016 1.1.0-phetio.4/ => @mattpen
drwxrwxr-x+ 4 tomcat     phet 118 Sep 24  2016 1.1.0-phetiodev.3/ => @samreid 
drwxrwsr-x+ 3 tomcat     phet 137 Aug 29  2016 1.1.0-dev.2/ => @samreid 
drwxrwsr-x+ 3 tomcat     phet 137 Aug 23  2016 1.1.0-dev.1/ => @samreid 
```

Questions:

- [ ] Why was this branch created? What PhET-iO work was involved?

- [ ] Are any of these versions used by clients, being maintained, in the PhET-iO spreadsheet, etc?

- [ ] If these versions were published from 1.1-phetio, why is the syntax for these version numbers all over the place? And I'm not clear on how we got version numbers with "1.1.x-dev.x" syntax when there is no 1.1 branch.

- [ ] 1.1.0-dev.3/ and 1.1.0-dev.4 have no associated commits. How could that be?

- [ ] I see git history for these 1.1 versions that do not appear on bayes. How could that be?

1.1.3-rc.2 => @zepumph 6/28/18
1.1.3-phetiorc.2 => @zepumph 6/28/18
1.1.3-rc.4 => @jonathanolson 6/29/18
1.1.3-rc.5 => @jonathanolson 7/6/18

**Comment by pixelzoom** [comment]
@kathy-phet said that this branch was created to support a study that Ian was doing. Record/playback was turned on to support that, but no other PhET-iO instrumentation was done.  And that I should just ignore the 1.1-phetio branch, and not be concerned about this for 1.2 publication in https://github.com/phetsims/function-builder/issues/139.

@samreid @zepumph anything to add to that?

**Comment by samreid** [comment]
Perhaps this overlaps in part with https://github.com/phetsims/function-builder/issues/100?

**Comment by zepumph** [comment]
That sounds right. We released a version a long long time ago outfitted with record/playback support for a researcher. As far as I know, it has not been used in years.

It is not actively maintenance released (from the PhET-iO deploy status in the admin section of the phet website).

You can search for "function" in [Partners.md](https://github.com/phetsims/phet-io/blob/dc4a8304ae89c353bd17bc4a034c04fb07d1efeb/doc/partners.md), you can find a couple more details.

**Comment by zepumph** [comment]
@jonathanolson was only involved in those releases as part of a batch MR, unassigning him.

**Comment by kathy-phet** [comment]
I don't think this issue needs more discussion. It's pretty much ancient history at this point, and @pixelzoom created 1.2 dev for phet brand. I'm going to close this issue.

**Comment by pixelzoom** [comment]
Thanks for the info.

I'm going to re-open this because the convention is to document branches with a GitHub issue. And the issue is to remain open until the branch is deleted.  In this case, it's questionable as to whether "1.1-phetio" is a non-standard branch name.  But as the responsible dev for this repo, I anticipate having questions about this branch in the future. So documenting this branch (and keeping this issue open) will prevent having this conversation again in the future.
